src/data-types/objects/creating-objects.js

```diff
@@ -1,7 +1,7 @@
 // TODO: write code in this section to pass the tests. You will need to add new code
 // as well as modify some of the existing code
-const person = null
-const computer = null
+const person = {name:"Jane", age:32}
```

Comment:
These objects might be a little more readable if you broke them down onto new lines. E.G
```js
{
  name: 'jane',
  age; 32
}
```